docs/source/conf.py

```diff
@@ -83,19 +83,27 @@ def __call__(self, directory):
 # zip the assets directory found in the examples directory and place it in the current dir
 def zip_dir(source_dir, zip_file_name):
     # Create archive with compressed files
-    with zipfile.ZipFile(file=TMP_FOLDER / zip_file_name, mode="w", compression=zipfile.ZIP_DEFLATED) as ziph:
+    with zipfile.ZipFile(
+        file=TMP_FOLDER / zip_file_name, mode="w", compression=zipfile.ZIP_DEFLATED
+    ) as ziph:
         for root, _, files in os.walk(source_dir):
             for file in files:
                 ziph.write(
                     os.path.join(root, file),
-                    os.path.relpath(os.path.join(root, file), os.path.join(source_dir, "..")),
+                    os.path.relpath(
+                        os.path.join(root, file), os.path.join(source_dir, "..")
+                    ),
                 )
 
 
-assets_dir_titanic = Path(__file__).parents[2] / "examples" / "titanic_example" / "assets"
+assets_dir_titanic = (
+    Path(__file__).parents[2] / "examples" / "titanic_example" / "assets"
+)
 zip_dir(assets_dir_titanic, "titanic_assets.zip")
 
-assets_dir_substrafl_fedavg = Path(__file__).parents[2] / "substrafl_examples" / "strategies_examples" / "assets"
+assets_dir_substrafl_fedavg = (
+    Path(__file__).parents[2] / "substrafl_examples" / "strategies_examples" / "assets"
+)
```

Comment:
please avoid reformatting this file automatically
not a big deal, but I think several people have different linter configs on their computer, and this file gets reformatted differently on a regular basis, that's bothersome